tests/jupyter/test_pythreejs.py

```diff
@@ -221,9 +232,10 @@ def test_labels():
     pl = pyvista.Plotter()
     pl.add_point_labels(poly, "My Labels", point_size=20, font_size=36)
 
-    # ensure that we ignore labels
-    with pytest.warns(UserWarning):
+    # ensure point labels at least don't raise a warning
+    with pytest.warns(None) as record:
         pv_pythreejs.convert_plotter(pl)
+    assert len(record) == 0
```

Comment:
Slight nitpick: `record` seems to have a `__bool__` that matches the length, so perhaps we could do
```suggestion
    assert not record
```
The current version is fine too, it's explicit.